app/scripts/controllers/metametrics.js

```diff
@@ -581,6 +584,28 @@ export default class MetaMetricsController {
     }, {});
   }
 
+  /**
+   *
+   * @param {*} metamaskState
+   * @returns number of unique collectible addresses
+   */
+  _getNumberOfNFtCollection(metamaskState) {
```

Comment:
I imagine this function can be a good candidate for lodash's `memoize` method. Can we add this in to ensure we do our best to optimize performance for our tracking methods?